lib/helpers/mail_sender.js

```diff
@@ -27,9 +27,19 @@ const buildMessage = (userEmail, subject, html) => {
   };
 };
 
-export const sendMail = async (userEmail, subject, html) => {
+const getTemplate = async (path, parametros) => {
+  return await HTMLUtils.getHTMLFile(`lib/html/${path}.html`, parametros);
+};
```

Comment:
Detalle: si es una función `async` de una línea que solo hace `return await`, conviene escribirla sin async/await:

```suggestion
const getTemplate = (path, parametros) =>
  HTMLUtils.getHTMLFile(`lib/html/${path}.html`, parametros);
```

Reply:
Explicación larga: https://jakearchibald.com/2017/await-vs-return-vs-return-await/

TLDR: `Outside of try/catch blocks, return await is redundant. There's even an ESLint rule to detect it, but it allows it in try/catch.`